app/assets/json/translations/react/es.json

```diff
@@ -195,11 +287,44 @@
+  "disclaimer.copyrightTitle": "Derechos de autor",
+  "disclaimer.intro": "Toda la información disponible en este sitio web se recopila, actualiza y suministra para la comodidad del lector. Si bien se toman todas las medidas razonables para que esta información esté actualizada y sea precisa, la Ciudad y el Condado de San Francisco no pueden garantizar la veracidad de la información que proviene de terceros.",
+  "disclaimer.liabilityP1": "La Ciudad y el Condado de San Francisco quedan eximidos de cualquier responsabilidad que surja de cualquier medida que se haya tomado o se haya dejado de tomar a partir de la información proporcionada por este medio, más allá de la fuente original. Asimismo, la Ciudad y el Condado de San Francisco no asumen ninguna responsabilidad relacionada con la confianza depositada en esta información.",
+  "disclaimer.liabilityTitle": "Responsabilidad",
+  "documentChecklist.assistedHousingDesc": "Necesitará una copia del contrato de alquiler que demuestre su domicilio actual.",
   "documentChecklist.assistedHousingTitle": "Preferencia para vivienda asistida",
+  "documentChecklist.copDoc1": "El Programa de Certificado de Preferencia (COP, siglas en inglés) está destinado a aquellas familias que resultaron desplazadas por la Agencia de Redesarrollo de San Francisco. De este modo, el programa COP les otorga prioridad en los sorteos de viviendas.",
+  "documentChecklist.copDoc2": "Obtenga más información sobre el <a href='https://sf.gov/learn-about-certificate-preference-cop' target='_blank'>Programa de Certificado de Preferencia</a>.",
+  "documentChecklist.dthpDoc1": "El Programa Preferencia de Vivienda para Inquilinos Desalojados (DTHP, siglas en inglés) está destinado a aquellas personas que perdieron sus viviendas a raíz de alguno de estos motivos:",
+  "documentChecklist.dthpDoc2": "<ul className='list-disc ml-7'><li>Ley Ellis o desalojo por mudanza del propietario</li><li>Daños mayores producidos por un incendio</li><li>Alquiler no asequible a causa del vencimiento de ciertas restricciones</li></ul>",
+  "documentChecklist.dthpDoc3": "Obtenga más información sobre el <a href='https://sf.gov/displaced-tenant-housing-preference-program-dthp' target='_blank'>Programa Preferencia de Vivienda para Inquilinos Desalojados</a>.",
   "documentChecklist.header": "Esta lista enumera los documentos que su grupo familiar podría tener que proporcionar al presentar una solicitud de vivienda.",
+  "documentChecklist.homebuyerDesc": "Necesitará los siguientes documentos para presentar su solicitud para las unidades en venta:",
+  "documentChecklist.homebuyerEducationDesc": "Comprobante de haber completado la capacitación para compradores de viviendas",
+  "documentChecklist.homebuyerHeader": "Para Compradores de Vivienda",
+  "documentChecklist.homebuyerLoanDesc": "Carta de pre-aprobación de préstamo hipotecario de un <a href='%{lenderListUrl}' target='_blank'>prestamista aprobado por MOHCD</a>",
```

Comment:
content mismatch with the English translation file (applies to other non-english translation files)